operate/wallet/master.py

```diff
@@ -393,6 +399,32 @@ def update_backup_owner(
 
         return False
 
+    @property
+    def enriched_json(self) -> t.Dict:
+        """Get JSON representation with extended information (e.g., safe owners)."""
+        rpc = None
+        wallet_json = self.json
+
+        if not self.safes:
+            return wallet_json
+
+        owner_sets = set()
+        for chain, safe in self.safes.items():
+            ledger_api = self.ledger_api(chain=chain, rpc=rpc)
+            owners = get_owners(ledger_api=ledger_api, safe=safe)
+            owners.remove(self.address)
+            wallet_json["safes"][chain.value] = {
+                wallet_json["safes"][chain.value]: {"owners": owners}
+            }
+            owner_sets.add(frozenset(owners))
+
+        wallet_json["all_chains_same_safe_address"] = len(set(self.safes.values())) == 1
+        wallet_json["all_safe_backup_owners_match"] = len(owner_sets) == 1
+        wallet_json["single_backup_owner_per_safe"] = all(
```

Comment:
maybe use instead:
`consistent_safe_address`  (= all chains same safe address)
`consistent_safe_owners` (= all chains safes = same set of owners  AND only 1 backup owner)
?